platform/wismote/dev/acc-sensor.c

```diff
@@ -42,8 +42,8 @@ static void
 activate(void)
 {
   /* This assumes that some other sensor system already did setup the ADC
-  /* (in the case of the sky platform it is sensors_light_init that does it)
-
+   * (in the case of the sky platform it is sensors_light_init that does it) */
+  #if 0
```

Comment:
Doxygen has problems with the not closed nested block comments. As far as I understand the current code, this block is also not active at the moment [platform/wismote/dev/acc-sensor.c#L42](https://github.com/contiki-os/contiki/blob/master/platform/wismote/dev/acc-sensor.c#L42) (Nested comments have the closing tag removed)
To fix the doxygen warnings I closed all comments with the respective end tag and added a `#if 0` to get the same behavior as before, as nested `/* */` comments don't work either (http://c-faq.com/misc/nestcomment.html).